.github/workflows/km-ci-workflow.yaml

```diff
@@ -254,11 +269,43 @@ jobs:
           name: km
           path: /tmp/
 
-      - name: Payload tests with packer
-        # to avoid calling packer in a loop, we call it on one dir (does not matter which one)
-        # and pass the parent to it, so that the scan of payloads will happen inside a single docker
-        run: make -C payloads/busybox test-withpacker PACKER_DIR=payloads TIMEOUT=15m STEP=test
-        # note: not using CI's 'timeout-minutes' to avoid a kill without cleaning resources
+      - name: Configure KRUN
+        run: |
+          sudo mkdir -p /opt/kontain/bin
+          sudo mv -t /opt/kontain/bin /tmp/km /tmp/krun
+          sudo chmod +x /opt/kontain/bin/km /opt/kontain/bin/krun
+          sudo bash ./container-runtime/podman_config.sh
+          sudo bash ./container-runtime/docker_config.sh
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666"' > /tmp/rules
```

Comment:
Should this be KERNEL="kvm"?

Single equal sign?

Reply:
== to compare, = to assign. So for kvm module (compare) assign group and permission.